client/src/components/JurisdictionAdmin/BatchRoundSteps/BatchRoundTallyEntry/BatchDetail.tsx

```diff
@@ -494,59 +507,66 @@ const BatchResultTallySheet: React.FC<IBatchResultTallySheetProps> = ({
         aria-labelledby={`${Classes.TAB_PANEL}_${batch.name}_${selectedTabId}`}
         role="tabpanel"
       >
-        <BatchResultTallySheetTable>
-          <thead>
-            <tr>
-              <th>Choice</th>
-              <th>Votes</th>
-            </tr>
-          </thead>
-          <tbody>
-            {contest.choices.map((choice, i) => (
-              <tr key={choice.id}>
-                <td>{choice.name}</td>
-                <td>
-                  {isTotalsSheet ? (
-                    <span>
-                      {sum(
-                        sheets.map(sheet => sheet.results[choice.id] ?? 0)
-                      ).toLocaleString()}
-                    </span>
-                  ) : isEditing ? (
-                    <input
-                      aria-label={`${choice.name} Votes`}
-                      // Should be fine accessibility-wise, having read and considered the
-                      // accessibility warning under
-                      // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus,
-                      // since we're auto-focusing after a relevant user action and the input is
-                      // clearly labeled
-                      // eslint-disable-next-line jsx-a11y/no-autofocus
-                      autoFocus={!isSelectedSheetNewAndUnsaved && i === 0}
-                      className={classnames(
-                        Classes.INPUT,
-                        errors.results?.[choice.id] && Classes.INTENT_DANGER
-                      )}
-                      name={`results[${choice.id}]`}
-                      readOnly={isSubmitting}
-                      ref={register({
-                        min: 0,
-                        required: true,
-                        valueAsNumber: true,
-                      })}
-                      type="number"
-                    />
-                  ) : (
-                    <span>
-                      {(
-                        selectedSheet?.results[choice.id] ?? ''
-                      ).toLocaleString()}
-                    </span>
-                  )}
-                </td>
+        {contests.map(contest => (
```

Comment:
Easier to review using https://github.com/votingworks/arlo/pull/1878/commits/7f0ad7ed9bbe5ab23a012d02dc5cb416e8b1c049?w=1